packages/components/Molecules/Search/searchtable.stories.js

```diff
@@ -0,0 +1,319 @@
+import RplSearchResultsTable from './SearchResultsTable'
+import { withKnobs, object } from '@storybook/addon-knobs/vue'
+
+export default {
+  title: 'Molecules/Search/RplSearchResultsTable',
```

Comment:
May just use `SearchResultsTable` instead of `RplSearchResultsTable ` to make it same as other storybook menu items.

Reply:
changed